submissions/AbrahamJudah/HTML/SignUp.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Registration Form</title>
+    <link rel="stylesheet" type="text/css" href= "styles.css">
+</head>
+<body>
+    <section>
+            <img src="vmart-logo.png" id="image" alt="logo" width="80" height="40" > <h1  class="header" >VivaMart</h1> 
```

Comment:
You need to push this picture, and include it this PR.
Also fix indentation on this line